test.lua

```diff
@@ -503,6 +503,10 @@ for idx,value in ipairs({ 1, 5 }) do
    assert(clusters[idx] == value)
 end
 
+local str = '中文标点符号测试：文字内容，９８满意,hello world'
+for p, ch in utf8.codes(str) do
+	print (ch, utf8.char(ch), utf8.ispunct(ch))
```

Comment:
Just printing an output isn't enough to *test* this, you need to actually have some test or assertion that proves the result is what it is expected to be for a given input.